bin/i18n-update.sh

```diff
@@ -0,0 +1,127 @@
+      echo "-p, --path    local path for generating files (by default a temp folder will be used)"
+      echo "-d, --debug   print extra info for debugging"
+      exit 0
+      ;;
+    -p|--path*)
+      shift
+      LOCAL_PATH=$1
+      shift
+      ;;
+    -d|--debug*)
+      shift
+      DEBUG='true'
+      ;;
+    *)
+      break
+      ;;
+  esac
+done
+
+function error() {
+  echo -e "\033[0;31m$1\033[0m"
+  exit 1
+}
+
+function arrayLength() { echo "$#"; }
+
+function fetch_translations() {
+  local plugin_name=$1
+  local output_path=$2
+  local used_strings_file=$3
+
+  echo -e "\n\033[1mDownload I18n translations for \"$plugin_name\" plugin\033[0m"
+  node gutenberg/packages/react-native-editor/bin/i18n-translations-download "$plugin_name" "$output_path" "$used_strings_file"
+
+  if [[ "$plugin_name" == "gutenberg" ]]; then
+    echo "Update \"react-native-editor\" package i18n cache"
+    cp -r "$output_path/gutenberg/data" gutenberg/packages/react-native-editor/i18n-cache
+    cp "$output_path/gutenberg/index.js" gutenberg/packages/react-native-editor/i18n-cache/index.native.js
+  fi
+}
+
+# Set target path
+if [[ -n "${LOCAL_PATH:-}" ]]; then
+  TARGET_PATH=$LOCAL_PATH
+else
+  TARGET_PATH=$(mktemp -d)
+  trap '{ rm -rf -- "$TARGET_PATH"; }' EXIT
+fi
+
+# Get parameters
+PLUGINS=( "$@" )
+
+echo -e "\n\033[1m== Updating i18n localizations ==\033[0m"
+
+# Validate parameters
+if [[ $((${#PLUGINS[@]}%2)) -ne 0 ]]; then
+  error "Plugin arguments must be supplied as tuples (i.e. domain path/to/plugin)."
+fi
+
+for (( index=0; index<${#PLUGINS[@]}; index+=2 )); do
+  PLUGIN_FOLDER=${PLUGINS[index+1]}
+
+  if [[ ! -d $PLUGIN_FOLDER ]]; then
+    NOT_FOUND_PLUGIN_FOLDERS+=( $PLUGIN_FOLDER )
+    echo -e "\033[0;31mPlugin folder \"$PLUGIN_FOLDER\" doesn't exist.\033[0m"
+  fi
+done
+if [[ $(arrayLength "${NOT_FOUND_PLUGIN_FOLDERS[@]+"${NOT_FOUND_PLUGIN_FOLDERS[@]}"}") -gt 0 ]]; then
+  exit 1
+fi
+
+# Set used strings target
+USED_STRINGS_PATH="$TARGET_PATH/used-strings.json"
+
+# Build Gutenberg
+echo -e "\n\033[1mBuild Gutenberg packages\033[0m"
+npm run build:gutenberg
+
+# Extract used strings for plugins
+METRO_CONFIG="metro.config.js" node gutenberg/packages/react-native-editor/bin/extract-used-strings "$USED_STRINGS_PATH" "${PLUGINS[@]}"
+
+# Download translations of plugins (i.e. Jetpack)
+TRANSLATIONS_OUTPUT_PATH="src/i18n-cache"
+for (( index=0; index<${#PLUGINS[@]}; index+=2 )); do
+  PLUGIN_NAME=${PLUGINS[index]}
+
+  fetch_translations "$PLUGIN_NAME" "$TRANSLATIONS_OUTPUT_PATH" "$USED_STRINGS_PATH"
+done
+
+# Download translations of Gutenberg
+fetch_translations "gutenberg" "$TRANSLATIONS_OUTPUT_PATH" "$USED_STRINGS_PATH"
+
+echo -e "\n\033[1mGenerating localization strings files\033[0m"
+
+# Generate localization strings files
+./bin/strings2android.js bundle/android/strings.xml "$USED_STRINGS_PATH"
+./bin/strings2swift.js bundle/ios/GutenbergNativeTranslations.swift "$USED_STRINGS_PATH"
```

Comment:
The previous scripts used here`po2android`/`po2swift` have been renamed to `strings2android`/`strings2swift` which matches the file we use for conversion (used strings JSON files).